<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>datarecord_base</class>
 <widget class="QWidget" name="datarecord_base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>powermeter（功率计）</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="datalist_pm"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>ps（功率源）</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="datalist_ps"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>sp（频谱仪）</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="datalist_sp"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>dcp（直流电源）</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="datalist_dcp"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>ana(网分)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="datalist_ana"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QComboBox" name="drb_modecomboBox">
             <item>
              <property name="text">
               <string>AUTO</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Manual</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="drb_record">
             <property name="text">
              <string>Record</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="drb_startautorecord">
             <property name="text">
              <string>start</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="drb_delaytime">
               <property name="text">
                <string>Delay:1000ms</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="drb_edit_delaytime"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>choiced(记录的数据）</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListView" name="datalist_choiced"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTextBrowser" name="acd_msg">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>ReCordTable</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="drb_clearnowrecord">
             <property name="text">
              <string>clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="OneListWidget" name="listWidget">
           <item>
            <property name="text">
             <string>123</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>33</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4124</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>124</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OneListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">onelistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
